third-party/lib/libgcc/Mybuild

```diff
@@ -1,7 +1,7 @@
 package third_party.lib
 
 static module libgcc_toolchain extends embox.lib.LibGcc {
-	@Rule(script="$(CP) \"$$($(CC) $(CFLAGS) -print-file-name=libgcc.a | sed $$'s/\r$$//')\" $(OBJ_DIR)/third-party/lib/libgcc")
+	@Rule(script="$(CP) \"$$($(LIBGCC_FINDER) -print-file-name=libgcc.a | sed $$'s/\r$$//')\" $(OBJ_DIR)/third-party/lib/libgcc")
```

Comment:
LIBGCC_FINDER defined in stm32_clang only, this will not work for other templates. As far as I understand, this line could be reverted?